upup/models/cloudup/resources/addons/coredns.addons.k8s.io/k8s-1.12.yaml.template

```diff
@@ -219,8 +179,8 @@ spec:
           failureThreshold: 5
         readinessProbe:
           httpGet:
-            path: /health
-            port: 8080
+            path: /ready
+            port: 8181
```

Comment:
this seems not to be working

checking and if needed we will also push a PR on upstream

Reply:
oh ok, needed `ready` plugin on configuration

fixed